lib/ruby-lsp-rails.rb

```diff
@@ -8,6 +8,10 @@
 require "ruby_lsp_rails/railtie"
 
 module RubyLsp
+  # # Supported features
+  #
+  # - [Hover](rdoc-ref:RubyLsp::Rails::Hover)
+  # - [CodeLens](rdoc-ref:RubyLsp::Rails::CodeLens)
```

Comment:
@vinistock I added this as an equivalent to https://shopify.github.io/ruby-lsp/RubyLsp/Requests.html, so we have somewhere to link do directly, does it seem suitable?

Reply:
Yeah, great idea. I think this is the place to put it.